tests/sles4sap/sapconf.pm

```diff
@@ -128,9 +129,10 @@ sub run {
     verify_sapconf_service('sysstat.service', 'Write information about system start to sysstat log')
       if (is_sle('>=15'));
 
-    my $statusregex = join('.+', @tuned_profiles);
     $output = script_output "tuned-adm list";
-    die "Command 'tuned-adm list' output is not recognized" unless ($output =~ m|$statusregex|s);
+    foreach my $p (@tuned_profiles) {
+        die "Profile $p is not in 'tuned-adm list'" unless (grep(/$p/, $output));
```

Comment:
@alvarocarvajald I changed the code a little bit because I first wanted to simplify the `@tuned_profiles` declaration to avoid duplicate profile names. I saw after that the order was in fact important for the regex (sounds logical).

But I think it's better to test each profile in a loop (can be a little bit slower but not that much) and this way if a profile name position is changed we don't have to change the code, WDYT?